src/components/App.js

```diff
@@ -0,0 +1,40 @@
+import { Layout } from 'antd';
```

Comment:
Is this file used somewhere? I can't see it imported anywhere

Reply:
On line 9 the `Content` and `Header` components are abstracted from `Layout`

Reply:
I was talking about the file itself, the `src/components/App.js` file, i don't see it imported/used anywhere, I must be missing something...

Reply:
`import DevConsoleApp from 'console/components/App';`, line 3 of `src/index.js`

Reply:
Right, I'm just blind. Sorry about the noise.